front/src/firebase/init.js

```diff
@@ -10,10 +10,12 @@ const firebaseConfig = {
     storageBucket: process.env.VUE_APP_FIREBASE_STORAGE_BUCKET,
     messagingSenderId: process.env.VUE_APP_FIREBASE_MESSAGING_SENDER_ID,
     appId: process.env.VUE_APP_FIREBASE_APP_ID,
+    measurementId: "G-R2TFC8JH1K"
```

Comment:
what is that ? can we have a env like above ?

Reply:
Oh yes my bad in move it to env file ....